weblab_cg/tests/test_chaste_cg.py

```diff
@@ -90,3 +93,90 @@ def test_missing_capacitance(self, tmp_path):
                                                 'pandit_model_2001_epi_old_no_capacitance')
         assert str(error.value) == \
             'Membrane capacitance is required to be able to apply conversion to stimulus current!'
+
+    @pytest.mark.chaste
```

Comment:
Would it make sense to have the script tests in a separate file?

Reply:
I guess that's a decent idea actually!